.travis.yml

```diff
@@ -608,6 +608,7 @@ install:
   - >
     ! grep 'ERROR: ' ~/.tox-venv-install.log .tox/*/log/*.log
 before_script:
-  - gem install inspec rubocop
+  - gem install inspec -v '~> 3'
```

Comment:
`gem` does not support inlining this from my testing. 

If anyone knows how to do it better, I am all ears.

`~>` means, bound before the next major version.

Reply:
One suggestion would be to use a Gemfile and bundler, which (imo) is a lot of extra complexity just to get this back down to one line. I think doing it on two separate lines is fine.

Reply:
Also, a little bit of interactive ruby to demo how the ~> comparator works for those that may not be familiar with it. In this case, `~> 3` expands to `>= 3, < 4`:

```rb
$ irb
irb(main):001:0> depcheck = Gem::Dependency.new('inspec', '~> 3')
=> <Gem::Dependency type=:runtime name="inspec" requirements="~> 3">
irb(main):002:0> depcheck.match?('inspec', '2.99')
=> false
irb(main):003:0> depcheck.match?('inspec', '3.0')
=> true
irb(main):004:0> depcheck.match?('inspec', '3.99')
=> true
irb(main):005:0> depcheck.match?('inspec', '4.0')
=> false
```